### PR TITLE
Added a better example of jenkins with cli

### PR DESCRIPTION
**Pull request by e-minguez**
As promised :)

It works 
image

### CODE REVIEW

File: jenkins/new-scan-engine/Jenkinsfile-sysdig-cli-scanner [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
+        }
+    }
+    stage('Scan image') {
+      steps {
+        withCredentials([usernamePassword(credentialsId: 'sysdig-secure-api-token', passwordVariable: 'SECURE_API_TOKEN', usernameVariable: '')]) {
+          sh '''
+            VERSION=$(curl -L -s https://download.sysdig.com/scanning/sysdig-cli-scanner/latest_version.txt)
+            curl -LO "https://download.sysdig.com/scanning/bin/sysdig-cli-scanner/${VERSION}/linux/amd64/sysdig-cli-scanner"
+            chmod +x ./sysdig-cli-scanner
+            ./sysdig-cli-scanner --apiurl https://eu1.app.sysdig.com docker://${DOCKER_REPOSITORY}
```
Comment (manuelbcd) [inline jenkins/new-scan-engine/Jenkinsfile-sysdig-cli-scanner:27]: may we add any comment suggesting to customize the endpoint/region?
https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges

just suggesting

File: jenkins/new-scan-engine/README.md [hunk 2]
```diff
@@ -30,4 +30,4 @@ The [example pipeline](Jenkinsfile-jenkins-plugin) shows how to use it to build
 Both approaches require a couple of things:
 
 * A valid Sysdig Secure API token
```
Comment (manuelbcd) [inline jenkins/new-scan-engine/README.md:32]: + "The appropriate Sysdig vulnerability scanning endpoint depending on your region"
https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges

**Review by manuelbcd — APPROVED** [review]
Approved. I just added a couple of suggestions for if they make sense to you

File: jenkins/new-scan-engine/Jenkinsfile-sysdig-cli-scanner (hunk 1 above)
Comment (e-minguez) [reply jenkins/new-scan-engine/Jenkinsfile-sysdig-cli-scanner]: Great suggestion! PTAL

File: jenkins/new-scan-engine/README.md (hunk 2 above)
Comment (e-minguez) [reply jenkins/new-scan-engine/README.md]: Thanks! I've made some changes, PTAL

**Comment by e-minguez** [comment]
@manuelbcd PTAL!

**Pull request merged**